### ISSUE TITLE
Underline and Strikethrough not reported in text in PDF files in Adobe reader

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#9677]
### Steps to reproduce:

Open a PDF file which contains underlined or text with "strikethrough".  (I've attached an example to this issue).

Have NVDA report font formatting or navigate through text with NVDA set to report font attributes.

[Here is strikethrough text.pdf](https://github.com/nvaccess/nvda/files/3260785/Here.is.strikethrough.text.pdf)

### Actual behavior:

NVDA reports italic and bold text, but not underline or strikethrough.

### Expected behavior:

NVDA should report underline and strikethrough in the same way as it reports bold or italic.

#### NVDA version:

NVDA 2019.2 beta 1 (installed)

#### Windows version:

Windows 10 version 18362.1 

#### Name and version of other software in use when reproducing the issue:
Adobe Reader DC 9.012.20034

#### Does the issue still occur after restarting your PC?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

Noted that resolution of #2410 claimed to fix both underline and strikethrough, although issue #2175 noted it was broken in Adobe Reader 10.1 and later, and I'm not sure which version was being used when 2410 was "fixed".

**Comment by ivnc** [comment]
With Adobe Reader 20.0 2020.013.20074 and NVDA 2020.4Beta3 I'm unable to read bold and/or italic neither with NVDA+F or with font attributes reporting enabled, in addition to underline and strikethrough, as originally reported.

Regards.

**Comment by ivnc** [comment]
One clarification: I've just seen that my issue is only reproducible when bold or italic text is in the same line that the rest of text, tested with a PDF document generated using Word. In case of underline and strikethrough the location of the affected text is irrelevant, it seems to be always reproducible.

**Comment by Adriani90** [comment]
Still reproducible with NVDA 2024.2. When openning the PDF in a browser like Firefox, only font size is reported, but things like font style or atributes are not reported at all.
